examples/express-sample/package.json

```diff
@@ -6,15 +6,15 @@
     "start": "DEBUG=express-sample:server node ./bin/www"
   },
   "dependencies": {
-    "body-parser": "^1.19.0",
-    "config": "^1.16.0",
-    "cookie-parser": "~1.3.3",
-    "debug": "^4.1.1",
-    "ejs": "~2.5.5",
-    "express": "^4.17.1",
+    "body-parser": "^1.20.2",
+    "config": "^3.3.11",
+    "cookie-parser": "~1.4.6",
+    "debug": "^4.3.4",
+    "ejs": "~3.1.10",
+    "express": "^4.19.2",
     "morgan": "^1.10.0",
-    "node-sass-middleware": "^0.11.0",
-    "raygun": "^0.8.0",
+    "node-sass-middleware": "^1.1.0",
+    "raygun": "file:../..",
```

Comment:
`raygun` dependency points to the one at the repo root, instead of using an outdated one from `npm`.